programs/debug_node/README.md

```diff
@@ -1,104 +0,0 @@
-
-Introduction
-------------
```

Comment:
We need to keep this document somewhere. Is it in the docs directory/sub-repository already?